.eslintrc.yaml

```diff
@@ -110,6 +110,9 @@ rules:
   }, {
     selector: "ThrowStatement > CallExpression[callee.name=/Error$/]",
     message: "Use new keyword when throwing an Error."
+    }, {
+    selector: "CallExpression[callee.object.name='assert'][callee.property.name='fail'][arguments.length=1]",
```

Comment:
Is there a valid use-case for a two argument `assert.fail`?

Also if `assert.fail(null, null, "message")` is a valid use-case, we should probably document that.